packages/EasyTest/src/InvalidDataMaker/InvalidDataMaker.php

```diff
@@ -0,0 +1,437 @@
+            (new Count([
+                'max' => $maxElements,
+            ]))->maxMessage,
+            [
+                '{{ limit }}' => $maxElements,
+            ],
+            $maxElements
+        );
+
+        yield from $this->create("{$this->property} has too many elements in the collection", $value, $message);
+    }
+
+    /**
+     * @return iterable<mixed>
+     */
+    public function yieldBlankString(): iterable
+    {
+        $value = '';
+        $message = $this->translateMessage((new NotBlank())->message);
+
+        yield from $this->create("{$this->property} is blank", $value, $message);
+    }
+
+    /**
+     * @return iterable<mixed>
+     */
+    public function yieldDateTimeLessThanOrEqualToNow(): iterable
+    {
+        $dateTime = Carbon::now();
+        $message = $this->translateMessage(
+            (new GreaterThan([
+                'value' => 'now',
+            ]))->message,
+            [
+                '{{ compared_value }}' => 'now',
+            ]
+        );
+
+        $value = $dateTime->clone()
+            ->subSecond()
+            ->toAtomString();
+        yield from $this->create("{$this->property} has less datetime", $value, $message);
+
+        $value = $dateTime->toAtomString();
+        yield from $this->create("{$this->property} has equal datetime", $value, $message);
+    }
+
+    /**
+     * @return iterable<mixed>
+     */
+    public function yieldEmptyArrayCollection(int $minElements): iterable
+    {
+        $value = new ArrayCollection();
+        $message = $this->translateMessage(
+            (new Count([
+                'min' => $minElements,
+            ]))->minMessage,
+            [
+                '{{ limit }}' => $minElements,
+            ],
+            $minElements
+        );
+
+        yield from $this->create("{$this->property} has too few elements in the collection", $value, $message);
+    }
+
+    /**
+     * @return iterable<mixed>
+     */
+    public function yieldIntegerGreaterThanGiven(int $lessThanOrEqualValue): iterable
+    {
+        $value = $lessThanOrEqualValue + 1;
+        $message = $this->translateMessage(
+            (new LessThanOrEqual([
+                'value' => $value,
+            ]))->message,
+            [
+                '{{ compared_value }}' => $lessThanOrEqualValue,
+            ]
+        );
+
+        yield from $this->create("{$this->property} has greater value", $value, $message);
+    }
+
+    /**
+     * @return iterable<mixed>
+     */
+    public function yieldIntegerGreaterThanOrEqualToGiven(int $lessThanValue): iterable
+    {
+        $value = $lessThanValue + 1;
+        $message = $this->translateMessage(
+            (new LessThan([
+                'value' => $value,
+            ]))->message,
+            [
+                '{{ compared_value }}' => $lessThanValue,
+            ]
+        );
+        yield from $this->create("{$this->property} has greater value", $value, $message);
```

Comment:
Let's keep a blank line before all the yield statements.

Reply:
I grouped yield and related data together in multy yield methods. Pls have a look on two variants:
      
        $message = $this->translateMessage((new Positive())->message);

        $value = -1;
        yield from $this->create("{$this->property} has negative value", $value, $message);

        $value = 0;
        yield from $this->create("{$this->property} has zero value", $value, $message);
`

      
        $message = $this->translateMessage((new Positive())->message);

        $value = -1;

        yield from $this->create("{$this->property} has negative value", $value, $message);

        $value = 0;

        yield from $this->create("{$this->property} has zero value", $value, $message);
`

Reply:
I grouped yield and related data together in multy yield methods. Pls have a look on two variants:
      
        $message = $this->translateMessage((new Positive())->message);

        $value = -1;
        yield from $this->create("{$this->property} has negative value", $value, $message);

        $value = 0;
        yield from $this->create("{$this->property} has zero value", $value, $message);
`

      
        $message = $this->translateMessage((new Positive())->message);

        $value = -1;

        yield from $this->create("{$this->property} has negative value", $value, $message);

        $value = 0;

        yield from $this->create("{$this->property} has zero value", $value, $message);
`